src/api/app/components/token_card_component.rb

```diff
@@ -0,0 +1,16 @@
+class TokenCardComponent < ApplicationComponent
+  def initialize(token:)
+    super
+
+    @token = token
+  end
+
+  def operation
+    "Operation: #{@token.class.token_name.capitalize}"
+  end
+
+  def token_package_link
+    link_to("#{truncate(@token.package.project.name, length: 80)}/#{truncate(@token.package.name, length: 80)}",
```

Comment:
So this link could be 161 characters long... this is too much. It's going to increase the card's width and the UI will look weird. It's also a waste of space. The package and project names should be truncated or elided.

Reply:
True, its very long, I set the truncation to 64 in total, so the same length that we allow for the token names